src/appleseed/foundation/utility/benchmark/benchmarksuite.cpp

```diff
@@ -257,51 +257,50 @@ void BenchmarkSuite::run(
                     BenchmarkSuite::Impl::measure_runtime_ticks,
                     measurement_count);
 
+            // Gather the timing results.
+            TimingResult timing_result;
+            timing_result.m_iteration_count = 1;
+            timing_result.m_measurement_count = measurement_count;
+            timing_result.m_frequency = static_cast<double>(stopwatch.get_timer().frequency());
+            timing_result.m_ticks = runtime_ticks > overhead_ticks ? runtime_ticks - overhead_ticks : 0.0;
+
+            // Post the timing result.
+            suite_result.write(
+                *this,
+                *benchmark.get(),
+                __FILE__,
+                __LINE__,
+                timing_result);
+
 #ifdef GENERATE_BENCHMARK_PLOTS
             vector<Vector2d> points;
 
-            for (size_t j = 0; j < 100; ++j)
+            const size_t PointCount = 100;
```

Comment:
To be consistent with other variable naming, do we rename ```PointCount``` into ```point_count``` ?

Reply:
We spell true constants (i.e. with a constant value, not just constant variables) in CamelCase.

Reply:
Okay